release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -109,6 +109,17 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see the xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[supported devices].
 
+[id="ocp-4-12-openshift-sdn-ovn-kubernetes-feature-migration-support"]
+==== OpenShift SDN to OVN-Kubernetes feature migration support
+
+If you migrate from the OpenShift SDN cluster network provider to the OVN-Kubernetes cluster network provider, you can migrate your configurations for the following capabilities:
+
+* Egress IP addresses
+* Egress firewalls
+* Multicast
+
+To enable this automatic migration for one or more of the aforementioned capabilities, enable the migration during the migration to OVN-Kubernetes. For more information, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc[Migrating from the OpenShift SDN cluster network provider].
```

Comment:
Similar to my comment above, this "enable the migration during the migration" confuses me. Enable the migration of what, during the migration to OVN-Kubernetes. Is there an option or something when you migrate to bring over these items?